spec/glimr_api_client/register_new_case_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+require 'support/shared_examples_for_glimr'
+
+RSpec.describe GlimrApiClient::RegisterNewCase do
+  include_examples 'register new case with glimr'
+
+  let(:params) { {} }
+	let(:excon) { class_double(Excon, post: double(status: 200, body: '')) }
+	let(:api) { described_class.new(params) }
+
+	before do
+		allow(api).to receive(:client).and_return(excon)
+	end
+
+  context 'validating parameters' do
+		let(:valid_params) { { jurisdictionId: 8, onlineMappingCode: 'something' } }
+
+		it 'barfs when no parameters are supplied' do
```

Comment:
Ok to use something other than 'barfs'?  it 'fails validation when no parameters...' maybe?  For me, 'barfs' is just a bit too colloquial.